src/render/index.html.eco

```diff
@@ -13,65 +13,109 @@ layout: default
         <h2>Connects <strong>CMS, CRM</strong> or other web platforms<br/>
           to create a <strong>unified customer experience.</strong></h2>
         <span class="github-btn">
-          <iframe src="https://ghbtns.com/github-btn.html?user=Cognifide&amp;repo=knotx&amp;type=star&amp;count=true&amp;size=large" frameborder="0" scrolling="0" width="160" height="30">
-          </iframe>
-        </span>
-        <span class="github-btn">
-          <iframe src="https://ghbtns.com/github-btn.html?user=Cognifide&amp;repo=knotx&amp;type=fork&amp;count=true&amp;size=large" frameborder="0" scrolling="0" width="160" height="30">
-          </iframe>
+          <p>
+            <a class="btn btn-default btn-lg" href="/download" role="button">
+              <span class="fa fa-download"></span>
+              Knot.x 1.3.0
+            </a>
+          </p>
         </span>
       </div>
     </div>
   </div>
 
-  <!-- Intro -->
-  <div class="feature intro">
+  <!-- diagram -->
+  <div class="diagram gray-background">
     <div class="container">
       <div class="row">
-        <div class="col-sm-6">
-          <div class="knotx-movie">
-            <iframe width="560" height="315" src="https://www.youtube.com/embed/dBhgsmC78Yw?rel=0" frameborder="0" allowfullscreen></iframe>
-          </div>
+        <div class="col-sm-1">
+          <h2>System Admin</h2>
+          <p>
+            manages the platform
+          </p>
         </div>
-        <div class="col-sm-6">
-          <blockquote>
-            <p>Let’s imagine an online banking website containing different features like a chat box, exchange rates, stock data and user profile information. The site has a high performance characteristic, despite its complexity and target publishing channels.
-              All those features come from different providers/vendors having their own teams working in various modes, technologies and release cycles.
-              Knot.x connects all of them in a controlled and isolated way, preventing any undesired interferences. It combines asynchronous programming principles and message-driven architecture providing a scalable platform for modern sites.
-              Knot.x connects all of the above to create a unified customer experience.
-            </p>
-          </blockquote>
+        <div class="col-sm-2">
+          <h2>Experience Manageable Integrations + Continuous Replatforming</h2>
+          <p>
+            combines various systems in a controlled and isolated way, preventing any
+            undesired interferences; gradually embed new systems, CMSes on your website
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Resilience</h2>
+          <p>
+            keeps your platform always available. Even if a module breaks, it can redeploy
+            themselves into a new location/server
```

Comment:
> Even if a module breaks, it can redeploy themselves into a new location/server

Should a module break, it can be redeployed automatically on a new server or in a new location, if necessary.